fearless/Common/Protocols/SheetAlertPresentable.swift

```diff
@@ -56,4 +76,34 @@ extension SheetAlertPresentable {
 
         controller.present(sheetController, animated: true)
     }
+
+    func present(
+        message: String?,
+        title: String,
+        closeAction: String?,
+        from view: ControllerBackedProtocol?
+    ) {
+        var currentController = view?.controller
+
+        if currentController == nil {
+            currentController = UIApplication.shared.delegate?.window??.rootViewController
```

Comment:
no good. can we create a new uiwindow to show alert? that's default apple practice